homeassistant/helpers/update_coordinator.py

```diff
@@ -85,15 +85,15 @@ def __init__(
         self._shutdown_requested = False
         if config_entry is UNDEFINED:
             # late import to avoid circular imports
-            from . import frame  # pylint: disable=import-outside-toplevel
+            from . import frame  # noqa: PLC0415
 
             frame.report_usage(
-                "relies on ContextVar, but should pass the config entry explicitly - see "
-                "https://developers.home-assistant.io/blog/xxxxxxx",
-                core_behavior=frame.ReportBehavior.LOG,
-                core_integration_behavior=frame.ReportBehavior.LOG,
-                custom_integration_behavior=frame.ReportBehavior.IGNORE,
-                breaks_in_ha_version="2026.3",
+                "relies on ContextVar, but should pass the config entry explicitly. "
+                "This will not be enforced for custom integrations - see "
+                "https://github.com/home-assistant/core/pull/138161#discussion_r1958184241",
```

Comment:
I don't think we need to include this in the log message (since we don't log custom integrations anyway), it should be a comment instead.